docs/source/test_network.md

```diff
@@ -239,34 +239,30 @@ chaincode on the channel using the following command:
 ```
 ./network.sh deployCC
 ```
-The `deployCC` subcommand will install the **fabcar** chaincode on
+The `deployCC` subcommand will install the **asset-transfer (basic)** chaincode on
 ``peer0.org1.example.com`` and ``peer0.org2.example.com`` and then deploy
 the chaincode on the channel specified using the channel flag (or `mychannel`
 if no channel is specified).  If you are deploying a chaincode for the first
 time, the script will install the chaincode dependencies. By default, The script
-installs the Go version of the fabcar chaincode. However, you can use the
+installs the Go version of the asset-transfer (basic) chaincode. However, you can use the
 language flag, `-l`, to install the Java or javascript versions of the chaincode.
-You can find the Fabcar chaincode in the `chaincode` folder of the `fabric-samples`
+You can find the asset-transfer (basic) chaincode in the `asset-transfer-basic` folder of the `fabric-samples`
 directory. This folder contains sample chaincode that are provided as examples and
 used by tutorials to highlight Fabric features.
 
-After the **fabcar** chaincode definition has been committed to the channel, the
+After the **asset-transfer (basic)** chaincode definition has been committed to the channel, the
```

Comment:
The deploy script no longer invokes the init function by default and adds asset to the ledger. You can pass that to the script. Perhaps better you can use that as an opportunity to introduce interacting with the network.

Reply:
Moved to 'interacting with the network' as suggested.